images/linux/scripts/installers/aws-sam-cli.sh

```diff
@@ -2,15 +2,31 @@
 ################################################################################
 ##  File:  aws-sam-cli.sh
 ##  Desc:  Installs AWS SAM CLI
-##         Must be run as non-root user after homebrew and clang
+##         Requires Pyton >=3.6, must be run as non-root user after toolset installation
 ################################################################################
 
 # Source the helpers for use with the script
 source $HELPER_SCRIPTS/document.sh
 
-# Install aws sam cli
-brew tap aws/tap
-brew install aws-sam-cli
+# Download latest aws sam cli sources
+TarballUrl=$(curl -s https://api.github.com/repos/awslabs/aws-sam-cli/releases/latest | jq -r '.tarball_url')
+TarballPath="/tmp/aws-sam-cli.tar.gz"
+wget $TarballUrl -O $TarballPath
+tar -xzvf $TarballPath -C /tmp
+SourcesDir=$(echo /tmp/awslabs-aws-sam-cli*)
+
+# Use python 3.7 from toolcache to install aws sam, setuptools package required for the installation
+Python3Dir=$(echo ${AGENT_TOOLSDIRECTORY}/Python/3.7*/x64)
+Python3BinDir=$(echo ${Python3Dir}/bin)
+export PATH="$Python3Dir:$Python3BinDir:$PATH"
+python3 -m pip install setuptools
+cd $SourcesDir
+python3 setup.py install
+sudo ln -sf ${Python3BinDir}/sam /usr/local/bin/sam
+
+# Cleanup downloaded files
```

Comment:
nitpick: we can remove this section and leave it the `cleanup.sh` script

Reply:
Cool, thanks!